test/e2e/e2e_test.go

```diff
@@ -486,6 +486,15 @@ func setUnsupportedServiceCAConfig(t *testing.T, config *rest.Config, forceRotat
 	if err != nil {
 		t.Fatalf("error retrieving operator config: %v", err)
 	}
+	var forceRotationReason string
+	i := 0
+	for {
+		forceRotationReason = fmt.Sprintf("service-ca-e2e-force-rotation-reason-%d", i)
+		if currentSigningKeySecret.Annotations[api.ForcedRotationReasonAnnotationName] != forceRotationReason {
+			break
+		}
+		i++
+	}
```

Comment:
just a little bit of syntax sugar sprinkles, also limiting the scope of `i`
```
var forceRotationReason string
for i := 0; ; i++ {
    forceRotationReason = fmt.Sprintf("service-ca-e2e-force-rotation-reason-%d", i)
    if currentSigningKeySecret.Annotations[api.ForcedRotationReasonAnnotationName] == forceRotationReason {
        break
    }
}
```

Reply:
Thanks, updated.